packages/mcfunction/src/colorizer/index.ts

```diff
@@ -18,8 +16,20 @@ export function register(meta: core.MetaRegistry) {
 		'mcfunction:command_child/trailing',
 		core.colorizer.error,
 	)
-}
-
-export const macro: core.Colorizer<CommandMacroNode> = (node, ctx) => {
-	return [core.ColorToken.create(node, 'string')]
+	meta.registerColorizer<MacroChildNode>(
+		'mcfunction:macro_child/macro',
+		macroChild,
+	)
+	meta.registerColorizer<MacroChildNode>(
+		'mcfunction:macro_child/other',
+		macroChild,
+	)
+	meta.registerColorizer<MacroChildNode>(
+		'mcfunction:macro_child/sign',
+		macroChild,
+	)
```

Comment:
would be nice if we had a `registerColorizers` where we could just pass in an array of node types

Reply:
Don't think that's in my department 😆